docs/installation/config/openzaak_config.rst

```diff
@@ -152,6 +152,25 @@ Open Notificaties
 
 All done!
 
+Register notification channels
+==============================
+
+Before notifications can be sent to ``kanalen`` in Open Notificaties, these ``kanalen``
+must first be registered via Open Zaak.
+
+In Open Zaak, activate the virtual environment:
+
+.. code-block:: bash
+
+    $ . env/bin/activate
+
+Then, for each ``kanaal`` described in ``open-zaak.<organization.local>/ref/kanalen/``, run the following
+command:
+
+.. code-block:: bash
+
+    (env) $ python src/manage.py register_kanaal <kanaal>
+
```

Comment:
I wouldn't let the end-user parse that documentation themselves. The channel name list is pretty static at this point, so imo we can include it here in the docs:

```suggestion
Register the required channels:

.. code-block:: bash

    $ python src/manage.py register_kanaal zaken
    $ python src/manage.py register_kanaal documenten
    $ python src/manage.py register_kanaal besluiten
    $ python src/manage.py register_kanaal autorisaties
    $ python src/manage.py register_kanaal zaaktypen
    $ python src/manage.py register_kanaal informatieobjecttypen
    $ python src/manage.py register_kanaal besluittypen
    

```